File: javascript/uri-online-judge/1012.js
const input = require('fs').readFileSync('/dev/stdin', 'utf-8');
const numbers = input.split(/\s/);
const values = numbers.map( item => Number(item) );

function triangleArea(base, height) {
  return  (base * height) / 2;
}

function circleArea(radius) {
  return 3.14159 * Math.pow(radius, 2);
}

function trapezeArea(baseMinor, baseMajor, height) {
  return ((baseMajor + baseMinor) * height) / 2
}

function retangleArea(sideA, sideB) {
 return sideA * sideB; 
}

const A = Number( values.shift() );
const B = Number( values.shift() );
const C = Number( values.shift() );

console.log(`TRIANGULO: ${ triangleArea(A, C).toFixed(3) }`);
console.log(`CIRCULO: ${ circleArea(C).toFixed(3) }`);
console.log(`TRAPEZIO: ${ trapezeArea(A, B, C).toFixed(3) }`);
console.log(`QUADRADO: ${ retangleArea(B, B).toFixed(3) }`);
console.log(`RETANGULO: ${ retangleArea(A, B).toFixed(3) }`);

File: c++/uri-online-judge/1012.cpp
#include <bits/stdc++.h>

using namespace std;

int main() {
  double a,b,c;
  cin >> a >> b >> c;

  cout << setprecision(3) << fixed << "TRIANGULO: " << ((a * c) / 2) << endl;
  cout << setprecision(3) << fixed << "CIRCULO: "   << (3.14159 * c * c) << endl;
  cout << setprecision(3) << fixed << "TRAPEZIO: "  << (((a + b) * c) / 2) << endl;
  cout << setprecision(3) << fixed << "QUADRADO: "  << (b * b) << endl;
  cout << setprecision(3) << fixed << "RETANGULO: " << (a * b) << endl;
}
